Turbulence.ModelGenerator/Preprocessing.cs

```diff
@@ -173,17 +173,42 @@ public static async Task ExtractTables(Uri inPath, Uri outPath)
     // Fix inconsistencies in downloaded files
     public static void PreExtract(Uri path)
     {
+        Console.WriteLine("Running pre-extract phase...");
+        
+        var inconsistentHeaderCount = 0;
+        var nameFieldCount = 0;
```

Comment:
just use bools, no need to count how often it happened, at least its not getting used rn